sky/cli.py

```diff
@@ -1595,7 +1624,8 @@ def autostop(
 @usage_lib.entrypoint
 # pylint: disable=redefined-builtin
 def start(clusters: Tuple[str], all: bool, yes: bool,
-          idle_minutes_to_autostop: int, retry_until_up: bool):
+          idle_minutes_to_autostop: Optional[int], terminate: bool,
```

Comment:
`sky start --terminate myclus` seems non-intuitive. IIUC it will restart the cluster then immediately shut it down. Wdyt about (1) leaving this flag out of this command; (2) keep this; (3) enforce if --terminate is set, -i must be set?

Reply:
Sounds great.